src/GovUk.Education.ExploreEducationStatistics.Admin/Services/PublicationService.cs

```diff
@@ -82,6 +82,22 @@ public class PublicationService : IPublicationService
                     }));
         }
 
+        public async Task<Either<ActionResult, List<TitleAndIdViewModel>>> ListPublications()
+        {
+            return await _userService
+                .CheckCanManageAllUsers()
```

Comment:
It was my intention to leave this endpoint as BAU only. The only reason I've moved it is because it's now used for both user management and on the Manage publication page - but in both cases its only used by BAU users.